Sources/Extensions/UIKit/UILabelExtensions.swift

```diff
@@ -18,6 +18,17 @@ public extension UILabel {
         self.text = text
     }
 
+    /// SwifterSwift: Initialize a UILabel with a text and font style.
+    ///
+    /// - Parameters:
+    ///   - text: the label's text.
+    ///   - style: the text style of the label, used to determine which font should be used.
+    public convenience init(text: String, style: UIFont.TextStyle) {
+        self.init()
```

Comment:
What would happen if you didn't call `self.init` and removed `convenience`? Would this still work?

Reply:
You cannot (or at least from what I know) create a designated initializer of UILabel in an extension.

Reply:
Ok checked now, the error is "Designated initializer cannot be declared in an extension of 'UILabel'; did you mean this to be a convenience initializer?"

So maybe a convenience initializer is better. Not sure is this is true with all types or just for types we don't "own" because declared in UIKit, because I don't recall having problems in creating initializers in extensions🤔